cico_setup.sh

```diff
@@ -135,20 +135,26 @@ function deploy() {
   local tag=$1
   local push_latest=$2
   local registry="push.registry.devshift.net"
-  
-  # build the deployable image
-  make docker-image-deploy
```

Comment:
It was moved to L147: https://github.com/fabric8-services/fabric8-wit/pull/2004/files#diff-66536a55e28a60c7e24d2d4a43a8b89cR147

The reason is because now we need to be logged in before doing the docker build